format/FormatNumber.js

```diff
@@ -218,7 +220,15 @@ function fmtNumberElement(v, opts = {}) {
         items.push(signGlyph(v, asElement));
     }
 
-    items.push(str);
+    if (isString(prefix)) {
+        if (str.startsWith('-') || str.startsWith('+')) {
+            items.push(str[0], '$', str.substring(1));
```

Comment:
Hi Yana.

I think here (and similarly below in `fmtNumberString`), we should ensure we are pushing the `prefix` string onto the `items` array. Here we are pushing a hardcoded `'$'` string, but we may want to use a different string for the prefix (e.g. `'£'`).

Reply:
Hi Tom,

Great catch!  I think I was staring at the ticket for too long and got confused that I hardcoded the `$` in.  😅

Lee just informed me that the tickets I'm working on seem to fall under your expertise, and I'd love to discuss and review code with you from now on (in addition to supervision from Lee).

As I'm still working on this ticket, I'd like to check in with you on a couple things:

1.  I'd like to get symmetry back into `formatNumber` in that we should not support something like: `▴+1.618` That `signGlyph`s should replace all signs (`+` and `-`), not only `-`.  And that this should be the case even if user explicitly sets `withPlusSign: true` What're your thoughts on this?

2. Since `prefix` is to come between a number and its sign, I'm wondering if it's better to separate a number and its sign into two variables instead of the combined `str`.